html/semantics/forms/the-form-element/form-requestsubmit.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<body>
```

Comment:
Missing `<title>`. Also, why add the `<body>` here (testharness.js usually goes in head)?

Reply:
Added <title>, removed <body>